addons/mail/static/src/discuss/call/common/call_menu.xml

```diff
@@ -4,14 +4,24 @@
     <t t-name="discuss.CallMenu">
         <div class="dropdown" t-attf-class="{{ className }}" t-ref="root">
             <button t-if="rtc.channel" class="user-select-none dropdown-toggle o-no-caret o-dropdown--narrow" t-att-title="buttonTitle" role="button" t-on-click="rtc.channel.open">
-                <div class="o-discuss-CallMenu-buttonContent d-flex align-items-center o-gap-0_5 rounded-3 opacity-75 opacity-100-hover px-1" t-att-class="{ 'o-is-talking': rtc.selfSession?.isActuallyTalking, 'o-isOdooCommunity': !isEnterprise }">
-                    <span class="position-relative small bg-transparent">
-                        <i class="me-2 fa fa-fw" t-att-class="icon" />
-                        <small class="d-flex position-absolute top-0 end-0 smaller bg-inherit">
-                            <i class="o-discuss-CallMenu-animation fa fa-volume-up o-discuss-inCallIconColor bg-inherit" t-att-class="{ 'o-isOdooCommunity': !isEnterprise }"/>
-                        </small>
-                    </span>
-                    <span class="text-truncate fw-bold pe-1" t-esc="rtc.channel.displayName"/>
+                <div class="o-discuss-CallMenu-buttonContent d-flex btn-group rounded-2 opacity-75 opacity-100-hover" t-att-class="{ 'o-is-talking': rtc.selfSession?.isActuallyTalking, 'o-isOdooCommunity': !isEnterprise }">
+                    <div class="o-discuss-CallMenu-channelInfo btn btn-group-item m-0 p-0 d-flex align-items-center o-gap-0_5 ps-1" role="button">
+                        <span class="position-relative small bg-transparent">
+                            <i class="me-2 fa fa-fw" t-att-class="icon" />
+                            <small class="d-flex position-absolute top-0 end-0 smaller bg-inherit">
+                                <i class="o-discuss-CallMenu-animation fa fa-volume-up o-discuss-inCallIconColor bg-inherit" t-att-class="{ 'o-isOdooCommunity': !isEnterprise }"/>
+                            </small>
+                        </span>
+                        <span class="text-truncate fw-bold pe-1" t-esc="rtc.channel.displayName"/>
+                    </div>
+                    <Dropdown>
```

Comment:
The truncate doesn't work:
<img width="361" height="40" alt="Image" src="https://github.com/user-attachments/assets/bf6f2c60-9698-4952-8f28-937560065fcd" />

If you put `text-truncate` on `o-discuss-CallMenu-channelInfo` then it works:
<img width="354" height="38" alt="Image" src="https://github.com/user-attachments/assets/27087e4c-0dbb-4f8d-ba7b-84e091e39432" />